Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -766,7 +791,7 @@ public struct ObjcLibrary: BazelTarget, UserConfigurable, SourceExcludable {
                 // it may not hold true for all cocoapods ( e.g. give it all
                 // possibilities here )
                 .named(name: "deps", value: deps
-                       .map { $0.filter { !$0.hasSuffix("_swift") } }
+                       .map { Array(Set($0)).filter { !$0.hasSuffix("_swift") } }
```

Comment:
Not `Set.filter`, or are you trying to coerce the result into an `Array`?

Reply:
Yep, we've gotta return a unique `Array` here - `Set.filter` returns an array.